_posts/2018-05-17-self-organization.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Self Organization"
+date: 2018-05-17 12:0:00 +0200
+categories: motivation how-we-learn
+author_name: Ugurcan Sengit
+author_url : /author/ugurcansengit
+author_avatar: ugurcansengit
+read_time : 8
+feature_image: posts/2018-04-20-holidaycheck-apprenticeship/apprenticeship.png
+---
+
+I remember the very first day as an apprentice at [HolidayCheck](https://twitter.com/holidaychecklab). Honestly, it's only four months ago, not so hard to recall. The thing is, it feels like it's been a lifetime already. All the things I've learnt during the apprenticeship... One just can't put a price tag on them. Time management and self-organization made them happen.
+
+At the very first day, I was both excited and afraid. Because I've never had six months of free space to improve myself after starting my professional career. The first challenge for me was to shape these six months. I've needed to create a structure and follow it strictly in order not to get lost during my journey. After my apprenticeship half-time presentation, I think I've nailed it, that's why I write this blog post to share my process with you.
+
+In my [3rd blog post](https://www.sengitu.com/posts/no-estimates/), I was so happy to announce my plan. After having that plan, the best thing I've done was to stick with it. I've shaped my plan over time to overcome problems I had, but I've never got out of the frame. Currently, in my [Trello board](https://trello.com/b/KlLdup7o/ugurcan-sengit-apprenticeship-board), I've following columns;
+
+* Backlog (One for each category)
+* Planned for This Week
+* Grooming
+* Ready To Do
+* In Progress
+* Done Weekly
+* Done Overall
+
+Each step has its own unique purpose. When I find a new subject to practice, I add it to the backlog. On each Monday, I have a planning meeting with my mentors, in that meeting, we pick topics to cover in the following week. I move these tasks to **Planned for This Week** column afterwards. After the meeting, I evaluate the tasks if they are small and descriptive enough. If not, big tasks go to **Grooming** column. In the **Grooming** column, I either investigate about the task or hack it quickly to understand if I can create smaller and descriptive pieces from that single big task. These new, small and descriptive tasks go to the **Ready To Do** column. In the **In Progress** column, I keep the task I am currently working on. During the day, you can check anytime to see what I am doing, that's what I call transparency. Ideally, there should be only one task active in the **In Progress** column. After I complete a task, I move them to **Done Weekly** column. Tasks wait here to be reviewed on the next planning meeting. After the weekly meeting, they are moved to **Done Overall** column. This column is analysed by the visualization tool, I will come to that.
```

Comment:
You could change "... meeting with my mentors, in that meeting" to "... meeting with my mentors, in which we pick topics..."

Reply:
Missing a "the" before "**Planned for This Week**"

Reply:
Same as above for the first "**Grooming**". It should be "... big tasks go to the **Grooming** column"

Reply:
In the last sentence, you could try "... by the visualization tool, which I'll come back to later".